public/locales/en-US/translations.json

```diff
@@ -222,8 +222,8 @@
   "offer_lack_of_funds": "The offer was partially filled, then cancelled due to lack of funds",
   "transaction_sequence": "The transaction's sequence number is",
   "trust_set_description": "It establishes <1><0>{{amount}}</0></1> as the maximum amount of <3><0>{{currency}}</0></3> from <5><0>{{issuer}}</0></5> that <7><0>{{account}}</0></7> is willing to hold",
-  "payment_desc_line_1": "The payment is from <1><0>{{account}}</0></1> to <3><0>{{destination}}</0></3>",
-  "the_source_tag_is": "The source tag is",
+  "payment_desc_line_1": "The payment is from <source/> to <destination/>",
+  "the_source_tag_is": "The source tag is ",
   "the_destination_tag_is": "The destination tag is",
   "payment_desc_line_4": "It was instructed to deliver",
```

Comment:
Is there a reason there's no `payment_desc_line_2` and line 3?

Reply:
No idea.

Reply:
Seems like in the code there are two other lines between them - so I guess those are the "lines 2" and "3" :P - Not important